test/challtestsrv.py

```diff
@@ -0,0 +1,269 @@
+import urllib2
```

Comment:
:tada:

Reply:
I thought you were really excited about urllib2, then realized this was a comment about the whole file. :joy: